tests/containers/podman_integration.pm

```diff
@@ -11,20 +11,52 @@ use Mojo::Base 'containers::basetest';
 use testapi;
 use serial_terminal qw(select_serial_terminal select_user_serial_terminal);
 use containers::utils qw(get_podman_version);
-use utils qw(zypper_call script_retry);
-use version_utils qw(get_os_release);
+use utils qw(zypper_call script_retry ensure_serialdev_permissions);
+use version_utils qw(get_os_release is_transactional is_sle is_sle_micro is_tumbleweed);
+use transactional qw(trup_call check_reboot_changes);
+use registration qw(add_suseconnect_product get_addon_fullname);
 use containers::common;
 
 sub run {
     select_serial_terminal;
     my ($running_version, $sp, $host_distri) = get_os_release;
     install_podman_when_needed($host_distri);
 
+    if (is_sle_micro) {
+        my $sle_version = "";
+        if (is_sle_micro('<5.3')) {
+            $sle_version = "15.3";
+        } elsif (is_sle_micro('<5.5')) {
+            $sle_version = "15.4";
+        } elsif (is_sle_micro('<6.0')) {
+            $sle_version = "15.5";
+        }
+        trup_call "register -p PackageHub/$sle_version/" . get_required_var('ARCH');
+        zypper_call "--gpg-auto-import-keys ref";
+    } elsif (is_sle) {
+        add_suseconnect_product(get_addon_fullname('phub'));
+    }
+
     # Install tests dependencies
-    my @pkgs = qw(apache2-utils bats buildah criu go gpg2 jq make netcat-openbsd openssl podman-remote python3-PyYAML skopeo socat sudo systemd-container);
-    zypper_call "in @pkgs";
+    my @pkgs = qw(bats jq make netcat-openbsd openssl python3-PyYAML socat sudo systemd-container);
+    push @pkgs, qw(apache2-utils buildah criu go gpg2 podman-remote skopeo) unless is_sle_micro;
+    if (is_transactional) {
+        trup_call "-c pkg install -y @pkgs";
+        check_reboot_changes;
+    } else {
+        zypper_call "in @pkgs";
+    }
+
+    # Some products don't have bernhard pre-defined (e.g. SLE Micro)
```

Comment:
Technically it's not the product but the way the tests are setup

Reply:
> Technically it's not the product but the way the tests are setup

Removed the comment.